ci/challenge_bins_test_settings.json

```diff
@@ -1,21 +1,37 @@
 {
     "timeout.seconds": "800",
     "tests.ignore": [
-        "challenge-3_amd64_program_go_patched.elf/output.json",
-        "challenge-3_amd64_program_go.elf/output.json",
-        "challenge-3_x86_program_go_patched.elf/output.json",
-        "challenge-3_x86_program_go.elf/output.json",
-        "challenge-3_arm64_program_go_patched.elf/output.json",
-        "challenge-3_arm64_program_go.elf/output.json",
-        "challenge-3_armv7_program_go_patched.elf/output.json",
-        "challenge-3_armv7_program_go.elf/output.json",
-        "challenge-3_arm64_program_go_patched.elf",
-        "challenge-3_arm64_program_go.elf",
-        "challenge-3_armv7_program_go_patched.elf",
+        "challenge-3_arm64_program_go_patched",
+        "challenge-3_arm64_program_go",
+        "challenge-3_armv7_program_go_patched",
+        "challenge-3_armv7_program_go",
+        "challenge-3_amd64_program_go_patched",
+        "challenge-3_amd64_program_go",
+        "challenge-3_x86_program_go_patched",
+        "challenge-3_x86_program_go",
         "challenge-3_armv7_program_go.elf",
-        "challenge-3_amd64_program_go_patched.elf",
-        "challenge-3_amd64_program_go.elf",
-        "challenge-3_x86_program_go_patched.elf",
-        "challenge-3_x86_program_go.elf"
+        "challenge-3_armv7_program_go_patched.elf",
+        "challenge-10-ppc-program_c.elf",
+        "challenge-10-ppc-vle-program_c.elf",
+        "ppc-adc_mpc5744p.elf",
```

Comment:
Should we be ignoring ppc here? I'd like to have at least one ppc-binary running in CI to merge this

Reply:
pushed a couple ppc hello binaries to the old test set
these were passing locally so they hopefully pass CI
challenge 10 ppc doesn't work atm